### PR TITLE
Education landing page

### PR DESCRIPTION
**Pull request by neynah**
Not ready to merge yet. Still waiting on updated use-case & number of users information from Grant Potter.

Feel free to review though! :)

### CODE REVIEW

File: go/education.html [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+      <h3>On-prem install for education privacy</h3>
+      <p>Run Sandstorm on hardware that you control. Everything users do on a Sandstorm server stays safely within your institution. Enjoy free dynamic DNS and SSL.</p>
+      
+    <li>
+      <h3>Self-service software</h3>
+      <p>Sandstorm provides simple web access to every user at the institution. When you approve Sandstorm, users do what they want safely within Sandstorm, and they file fewer support tickets.</p>
+
+    <li>
+      <h3>Single-sign-on and web access</h3>
+      <p>Once you enable login via Google, GitHub, or email, every app automatically uses that login provider. LDAP and SAML are to arrive soon.</p>
```
Comment (kentonv) [inline go/education.html]: LDAP and SAML work now. :)

File: style.scss [hunk 2]
```diff
@@ -2835,3 +2835,397 @@ body >footer {
...
+    
+    >h3 {
+      max-width: 940px;
+      margin: 1em auto;
+    }
+
+    ul {
+      padding: 0;
+      
+      li {
```
Comment (kentonv) [inline style.scss]: Consider adding `vertical-align: top` to this element so that side-by-side item titles line up.

File: go/education.html [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+    
+    <li>
+      <h3>Custom apps can use the built-in access control</h3>
+      <p>Academic departments need custom apps. As those apps get older, Sandstorm helps you sleep well at night. Sandstorm automatically mitigates against <a href="https://docs.sandstorm.io/en/latest/using/security-non-events/">95% of app security issues, according to our analysis.</a></p>
```
Comment (kentonv) [inline go/education.html]: "mitigates against" -> "mitigates"

Or you could say "protects against" or "defends against" if you prefer.

File: go/education.html [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+    
+  </ul>
+  <p class="highlight">Learn more about <a href="/how-it-works">Sandstorm's security model</a>.</p>
+</section>
+
+<section id="UNBC">
+<h2>See how UNBC uses Sandstorm</h2>
+  <div class="content">
+  <div class="text">
+  <p>Grant Potter at the University of Northern British Columbia runs a server supporting 70+ faculty and staff and writes tutorials to help people learn Piwik and other tools. He also uses Sandstorm apps to take meeting minutes. </p>
```
Comment (kentonv) [inline go/education.html]: It feels a little funny to mention "meeting minutes" which sounds like a very small thing, but I don't have a better suggestion.

Comment (paulproteus) [inline go/education.html]: FWIW IMHO meeting minutes are one of the main documents I create in some
jobs, so I do think it is a big thing!​

File: style.scss (hunk 2 above)
Comment (neynah) [inline style.scss]: Fixed, thanks.

File: go/education.html (hunk 3 above)
Comment (neynah) [inline go/education.html]: I used "Protects against"

**Comment by neynah** [comment]
Ready for review!

**Comment by paulproteus** [comment]
My feedback on this version.

Showstoppers IMHO:

- "BC" is undefined in the testimonials area, and so it's not clear what that means if you aren't Canadian. To define it, here is some proposed text for the header instead of "See how BC educators use Sandstorm"
    - Replace "See how BC educators use Sandstorm" with "Here's what British Columbia technologists love about Sandstorm"
    - "See how educators in British Columbia use Sandstorm"

Non-showstoppers:

- I expected the icon in "Publish online with WordPress" to be an appdemo link, not just the word WordPress. Is that expected? If the image won't be a link, then I would make these link to the app market, rather than link to appdemo URLs.

- The "See how BC educators use Sandstorm" section doesn't show me _how_ they use Sandstorm. It does show why.
    - Perhaps adding "Read more &raquo;" to the end, which links to the same place <http://bcopened.org/>, would solve the problem.

- I wish the "burdened by layers of required administrative and technical oversight" text were bigger, or earlier in the testimonial. It is the best part. Making the whole quote bigger would help.

Thanks again for requesting my feedback. I'm excited for this to go live and then work on publicity.

**Comment by neynah** [comment]
Fair enough. Thanks for the suggestions. I'll take a look & make some revisions within a couple hours. :)

**Comment by neynah** [comment]
If there were a read more like perhaps it could go here: http://bcopened.org/?s=oet.sandcats. Alternatively, I could also grab another quote from Grant Potter's testimonials that perhaps describes how they use Sandstorm. Since it's not a showstopper I've left it out for now.

- The BC testimonials section now says "See how educators in British Columbia use Sandstorm."
- I've adjusted the layout so that it now fits the quote with a larger font size.
- Links go to app market

**Comment by paulproteus** [comment]
This looks great! One issue:

- When I click "Start Sandstorm setup" on the top, or the bottom, I go to https://oasis.sandstorm.io/shared/RvA7wR5iE6Jef0NYJTI_lVtnXyHjG2v3FR3o8FBs8dI which appears to be a QuickSurvey survey. Instead, I think we agreed it should go to the feature key tool: https://sandstorm.io/get-feature-key . Is that a typo, or a new decision?

Beyond that, super great. I like the big Grant Potter, personally.

**Comment by neynah** [comment]
Typo, thanks. Fixed. :)

**Pull request closed without merging**